scripts/check_frontend_coverage.py

```diff
@@ -342,9 +342,11 @@
     'suggestion-modal-for-exploration-player.service.ts',
     'summary-list-header.directive.ts',
     'supplemental-card.directive.ts',
+    'svg-sanitizer.service.ts',
```

Comment:
Per the instruction at the top of this list, please do not add new files to it. All files should have full coverage.

Reply:
Most of the code in this file was taken from Image-upload-helper which was not fully covered in the first place. There are two `if/ else` statements that were not covered in the previous file that the code was borrowed from. I will try to fix those cases but for now, I haven't.

Reply:
OK, still needs addressing. Please talk to @nithusha21, he decides policy on this one.

Reply:
I generally only allow this if it is refactored from an existing file which was on the list. So I think this is ok.